tools/taskcluster/execute-bitbar-test.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script initiates a UI test-run after a upload of both the application and test
+APKs to Bitbar Cloud.
+"""
+
+from testdroid import Testdroid
+import json
+import sys
+import time
+
+with open('../../.bitbar_token.json') as json_file:
+    token_data = json.load(json_file)
+
+testdroid = Testdroid(apikey=token_data['bitbarToken'], url=token_data['cloudURL'])
+
+if len(sys.argv) != 3:
+    print "Bitbar Cloud: Application and Test Application APK path required."
+    exit(1)
+
+app_apk = testdroid.upload_file("../../app/build/outputs/apk/%s/%s.apk" % (sys.argv[1], sys.argv[2])) # Application
+test_app_apk = testdroid.upload_file("../../app/build/outputs/apk/androidTest/%s/%s-androidTest.apk" % (sys.argv[1], sys.argv[2])) # Test Application
+
+# Custom configuration using preset: android framework, project & device group
+test_run = testdroid.start_test_run_using_config(json.dumps({
+    "deviceGroupId": 42,
+    "files": [{"id": app_apk['id']}, {"id": test_app_apk['id']}],
+    "frameworkId": 24,
+    "limitationType": "CLASS",
+    "limitationValue": "org.mozilla.tv.firefox.ui",
+    "osType": "ANDROID",
+    "projectId": 296208,
+    "scheduler": "PARALLEL",
+    "testRunName": str(time.strftime("%m/%d/%Y %H:%M:%S"))
+}))
+
+print "Bitbar Cloud: UI Test Run Created - " + str(test_run['id'])
+
+# Automatically polls for completion of a test run
+testdroid.wait_test_run(test_run['projectId'], test_run['id'])
+
+# Check for test run failures and exit on test failure
+test_result = testdroid.get_test_run(test_run['projectId'], test_run['id'])
+
+if test_result['failedTestCaseCount'] != 0:
+    print "Bitbar Cloud: UI Test(s) have failed. Please check #bitbar-firefox-tv in Slack for URL."
```

Comment:
Since we already have the number of failures, could we include that in this log?  `$failedCount UI Test(s) have failed.`